### DISCUSSION TITLE
Add Brevo intergration to Elementor Forms

### DISCUSSION THREAD

**Discussion by CarterStevens1** [elementor/elementor#21607] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Will allow integration with send in blue contact lists and resolve the use for another plugin.

### Describe the solution you'd like

If Brevo intergration could be added to the "actions after submit" block to allow the choice of which contact list to add the user to!

**Comment by hitsgee** [comment]
This was a big issue for me ever since MailChimp got pricey.

However there is a simple free plugin to add this functionality and more here https://wordpress.org/plugins/integration-for-elementor-forms-sendinblue

The developer is a great guy too who listens to feedback and is always improving.

I would still love this to be native but the plugin works great for now.

Hope it helps.

**Reply by khammileon** [reply]
💯 Second that!  

Almost all my clients are using Brevo (formerly SendInBlue) for their mail marketing needs. That would be awesome and would avoid installing an additional extension such as MailOptin.

**Reply by CapEric** [reply]
Ali Addons is one of choices. Many integrations and extensions for Elementor Form widget. https://aliaddons.com/

**Comment by shelidodson** [comment]
I concur! Brevo is a CRM platform and has a plug-in, so it seems ridiculous that we have to get a plugin from another developer to integrate properly.
Please fix this ASAP. I have spent HOURS searching the help pages and internet because I was convinced this could be done with just hte Brevo SMTP plug in. Very disappointed in Brevo thus far. This is just one more way it is not intuitive or user-friendly.

**Comment by KhalilGit** [comment]
We really need this feature, just like the one you provided for Mailchimp. Please provide the Elementor-built-in integration for the Brevo platform.
